docs/entity_alarm_control_panel.md

```diff
@@ -11,15 +11,39 @@ sidebar_label: Alarm Control Panel
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| code_format | string | `None` | The format of the panel code.
+| state	| string | **Required** | One of the states listed in the states section.
+| code_format | string | `None` | Regex for code format or `None` if no code is required.
```

Comment:
Right now the only thing that the frontend supports is "number"

Reply:
I stole the info from the doc-string of __init__.py. Perhaps we should start with updating that part then.